src/FamilyHubs.ServiceDirectory.Core/Postcode/Model/PostcodeInfo.cs

```diff
@@ -1,21 +1,35 @@
-﻿using System.Text.Json.Serialization;
+﻿using Newtonsoft.Json;
 
 namespace FamilyHubs.ServiceDirectory.Core.Postcode.Model;
 
 // slightly leaky re: clean architecture, but keeps things simple and optimised
 
-/// <param name="Postcode">Searched postcode in canonical format. 2, 3 or 4-character outward code, single space and 3-character inward code.</param>
```

Comment:
Why have the [structured comments](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/xmldoc/) been removed?

They're useful as they provide intellisense prompts to the developer.